nextflow/beamform_module.nf

```diff
@@ -15,8 +15,6 @@ params.channels = null
 params.vcstools_version = 'master'
 params.mwa_search_version = 'master'
 
-params.basedir = '/group/mwavcs/vcs'
```

Comment:
I'm assuming that taking these two lines out completely is ok because it defaults back to the global config settings?

Reply:
In theory yes, but these aren't actually set in the config file currently. They are also overridden by the command-line arguments.

Reply:
I made those changes though :)

Reply:
If they're not in the config file, how would it work "in theory" then?

Reply:
I misread your first comment is how :sweat_smile:

Reply:
lol :+1: